app/src/main/java/com/door43/translationstudio/core/TargetTranslation.java

```diff
@@ -177,6 +177,27 @@ public String getTargetLanguageId() {
     }
 
     /**
+     * apply new format to translation
+     * @param translationFormat
+     */
+    public void applyFormat(TranslationFormat translationFormat) {
+        manifest.put("format", translationFormat.toString());
+    }
+
+    /**
+     * get format of translation
+     * @return
+     */
+    public TranslationFormat getFormat() {
+        String formatStr = manifest.getString("format");
```

Comment:
Since this won't change I'd move the loading of this information into the constructor so you don't have to do file io every time this is called.